operations-stdlib/src/commonMain/kotlin/string/Trim.kt

```diff
@@ -0,0 +1,54 @@
+package string
+
+import operation.StandardLogicOperation
+import utils.asList
+
+object Trim : StandardLogicOperation, StringUnwrapStrategy {
+    private const val ARGUMENTS_ARG_INDEX = 0
+    private const val TEXT_ARG_INDEX = 0
+    private const val CHAR_ARG_INDEX = 1
+    private const val MODE_ARG_INDEX = 2
+
+    override fun evaluateLogic(expression: Any?, data: Any?): Any? =
+        expression.asList.toOperationArguments()?.invokeTrim()
+
+    private fun TrimArguments.invokeTrim() = (this as? TrimArguments)?.modeBasedTrim(mode, char)
```

Comment:
To improve readability move this function under all of the rest

Reply:
Done